recipes/oatpp-swagger/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 2.8.12)
```

Reply:
This is oatpp's library and that package requires version [3.1](https://github.com/oatpp/oatpp/blob/a919b078edc87583f8f56227e80e7ed045356173/CMakeLists.txt#L1) so library for this project shouldn't be build on fewer version I think.

Reply:
The version of this cmake project (cmake_wrapper) and the version of upstream cmake project are not related. We only require a version of our `CMakeLists.txt` to be 2.8.x. 

Do not confuse with the version of `test_package/CMakeLists.txt` which should be 3.1 (enforced by [hooks](https://github.com/conan-io/hooks))

Reply:
Ok